.github/workflows/c3-e2e.yml

```diff
@@ -75,6 +75,46 @@ jobs:
           git config --global user.email wrangler@cloudflare.com
           git config --global user.name 'Wrangler Tester'
 
+      - name: Deploy test worker
+        if: steps.changes.outputs.everything_but_markdown == 'true' && env.TEST_CLOUDFLARE_ACCOUNT_ID != ''
+        shell: bash
+        run: |
+          # Create temporary directory for worker files
+          mkdir -p /tmp/existing-script-worker
+
+          # Create worker script
+          cat > /tmp/existing-script-worker/index.js << 'EOF'
+          export default {
+            async fetch(request, env, ctx) {
+              return new Response('Hello World!');
+            },
+          };
+          EOF
+
+          # Create wrangler.jsonc with FOO variable
+          cat > /tmp/existing-script-worker/wrangler.jsonc << 'EOF'
+          {
+            "name": "existing-script-test-do-not-delete",
+            "main": "index.js",
+            "compatibility_date": "2023-05-18",
+            "vars": {
+              "FOO": "bar"
+            }
+          }
+          EOF
+
+          # Check if worker exists and deploy only if it doesn't
+          cd /tmp/existing-script-worker
+          if ! npx wrangler@latest list | grep -q "existing-script-test-do-not-delete"; then
```

Comment:
Unfortunately this `wrangler list` command does not exist.
Perhaps instead we can just try curling the deployed URL 
https://devprod-status-bot.devprod.workers.dev/ and see if we get a 200 response